File: app/src/main/java/com/codylimber/fieldphenology/data/api/ApiModels.kt
package com.codylimber.fieldphenology.data.api

data class PlaceResult(val id: Int, val name: String, val adminLevel: Int? = null)

data class TaxonResult(
    val id: Int,
    val displayName: String,
    val scientificName: String,
    val commonName: String,
    val rank: String
)

File: ios/Manakin/Data/Models/ApiModels.swift
import Foundation

struct PlaceResult: Identifiable, Equatable, Hashable {
    let id: Int
    let name: String
    let adminLevel: Int?

    init(id: Int, name: String, adminLevel: Int? = nil) {
        self.id = id
        self.name = name
        self.adminLevel = adminLevel
    }
}

struct TaxonResult: Identifiable, Equatable, Hashable {
    let id: Int
    let displayName: String
    let scientificName: String
    let commonName: String
    let rank: String
}
